src/commands/indexer/index.ts

```diff
@@ -395,14 +395,7 @@ export default class Indexer extends HealthCheck {
                 `handleMintEvent ${networks[job.network].explorer}/tx/${transaction.hash}`,
                 tags,
               )
-              await handleMintEvent.call(
-                this,
-                this.networkMonitor,
-                transaction,
-                job.network,
-                tags,
-                this.updateMintedERC721,
-              )
+              await handleMintEvent.call(this, this.networkMonitor, transaction, job.network, tags)
```

Comment:
@BravoNatalie @alexanderattar I also think we should have the original handler and the new handler that submits a job. That way we can transition from on version to anther. Aka `handlers/handle-mint-events.ts` is the original and the new `handler/handle-mint-eventsv2.ts` that only submits a job. Or we just make a new generic job sender function that is called here. In short, what would be a good method to follow to transition into the new setup.